ios-contact-manager/ios-contact-manager/Controller/EditViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  EditViewController.swift
+//  ios-contact-manager
+//
+//  Created by Swain Yun on 10/10/23.
+//
+
+import UIKit
+
+final class EditViewController: UIViewController {
```

Comment:
```suggestion
final class ContactEditViewController: UIViewController {
```
역할이 좀 더 분명히 보이도록 `ContactEditViewController` 로 이름이 지어도 될 것 같아요 ㅎㅎ
마찬가지로 TableViewController 가 연락처 리스트 를 담은 뷰컨트롤러이니 TableViewController 도 ContactListViewController 로 이름 지어도 될 것 같아요 👍

Reply:
현재 EditViewController 는 연락처 생성(create) 과 수정(edit) 을 모두 해주고 있습니다.
뷰 컨트롤러를 두 개로 나누어서 한 객체는 생성만, 한 객체는 수정만 하는 화면을 만들어보는 건 어떨까요? 
그리고 이 과정에서 중복을 피하기 위해 상속을 사용하시는 것도 좋고요 🙂

Reply:
앞서 디스코드에서 말씀해주신 것의 연장선으로, 현재 연락처 정보를 생성하는 뷰 컨트롤러와 수정하는 뷰 컨트롤러로 분리하였습니다.
그러나 스토리보드 상의 뷰를 하나만 가지고 돌려쓰려고 했는데, 스토리보드 방식으로는 어떻게 할 수가 없었습니다.
일단 동일한 뷰를 하나 더 그려서 각자 StoryboardIdentifier를 가지고 사용하고 있는데, 추후 스텝3에서 다시 살펴볼 예정입니다.